addons/point_of_sale/models/pos_order.py

```diff
@@ -362,7 +358,9 @@ def _default_pricelist(self):
     nb_print = fields.Integer(string='Number of Print', readonly=True, copy=False, default=0)
     pos_reference = fields.Char(string='Receipt Ref', readonly=True, copy=False)
     sale_journal = fields.Many2one('account.journal', related='session_id.config_id.journal_id', string='Sale Journal', store=True, readonly=True)
-    fiscal_position_id = fields.Many2one('account.fiscal.position', string='Fiscal Position', default=lambda self: self._default_session().config_id.default_fiscal_position_id)
+    fiscal_position_id = fields.Many2one('account.fiscal.position', string='Fiscal Position', readonly=True, states={'draft': [('readonly', False)]},
+                                         default=lambda self: self._default_session().config_id.default_fiscal_position_id)
+    fiscal_position_ids = fields.Many2many("account.fiscal.position", related="session_id.config_id.fiscal_position_ids", string="Fiscal Positions")
```

Comment:
Why storing several fiscal position on a pos.order ? In case you sell to several customers on the same sale order ? Seems useless.